warehouse/query-core/src/main/java/datawave/query/attributes/Geometry.java

```diff
@@ -102,10 +102,12 @@ public int compareTo(Geometry other) {
         if (geometry == null) {
```

Comment:
[ERROR] Possible null pointer dereference of Geometry.geometry in datawave.query.attributes.Geometry.compareTo(Geometry) [datawave.query.attributes.Geometry, datawave.query.attributes.Geometry, datawave.query.attributes.Geometry] Dereferenced at Geometry.java:[line 108]Null value at Geometry.java:[line 102]Known null at Geometry.java:[line 103] NP_NULL_ON_SOME_PATH